src/Skeletor/Packages/BehatBehatPackage.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Skeletor\Packages;
+
+use Skeletor\Frameworks\Framework;
+
+class BehatBehatPackage extends Package
+{
+    public function setup()
+    {
+        $this->setInstallSlug("behat/behat");
+        $this->setName("Behat Behat");
+    }
+
+    public function configure(Framework $activeFramework)
```

Comment:
It case the best practise would be to have to interfaces, PackageInterface and ConfigurablePackageInterface that extends from PackageInterface. So you can pick the one you need for each case.